### DISCUSSION TITLE
Joplin server

### DISCUSSION THREAD

**Discussion by supperdd** [tteck/Proxmox#1959] [Ideas]
Everything is said in the title,
it's a notes manager

THANKS

**Comment by AndreaRicci94** [comment]
+1 for this, that would be really appreciated!

**Reply by mundeczek** [reply]
Joplin server is available in Runtipi LXC.
Working very well.

**Reply by AndreaRicci94** [reply]
Hi there,
That is true but the beauty of tteck's work is that you don't add any unnecessary overhead such as Docker or other "management" systems like Runtipi is.

I would be very happy if a solution for installing Joplin server "bare metal" on a Proxmox LXC became available....

**Reply by sunfire-design** [reply]
+1 for Joplin :)

**Reply by madhur** [reply]
+1 for this

**Reply by DimonXe** [reply]
+1 for Joplin

**Reply by tteck** [reply]
Joplin is primarily a [desktop application](https://joplinapp.org/help/install/), however, in my opinion, its [terminal application](https://joplinapp.org/help/apps/terminal/) is unattractive. Joplin-Server is Docker only.

**Reply by DimonXe** [reply]
It's sad.
I've never found alternatives in years. There is no selfhosted note manager that allows you to share notes.

**Reply by manuelkue** [reply]
@DimonXe I'm looking forward to trying out the selfhosted variant of [Anytype](https://anytype.io/):
https://tech.anytype.io/how-to/self-hosting


Seems a bit more difficult but should possibly this year also allow sharing notes and pages.

**Reply by kiwinh0** [reply]
+1

**Reply by protonaut** [reply]
+1
